### DISCUSSION TITLE
vultr使用mkcp传输端口/ip迅速被封锁

### DISCUSSION THREAD

**Discussion by lu1kaifeng** [v2fly/v2ray-core#761] [Q&A]
河南电信，从三月初，用vultr vps搭建的vmess kcp，莫名被封锁ip了，更换ip后，4小时左右再次被封，前几天开始封端口而不封ip了，是否墙加高了？

**Comment by lu1kaifeng** [comment]
或者就是说现在墙对这种不明的udp，只要检测到就封ip，不知道是否有人这段时间经历过

**Reply by xiaokangwang** [reply]
不发具体配置不好评估呀，开seed了么？看一下是在哪一跳丢包的？
